helm/alfresco-content-services/templates/pvc.yaml

```diff
@@ -22,3 +22,7 @@ spec:
 {{- if and (not .Values.filestore.persistence.existingClaim) (eq .Values.repository.edition "Enterprise") .Values.filestore.persistence.enabled }}
```

Comment:
There's something wrong with your PR. The diff should not look like that given current status of the [`pvc.yaml` in master](https://github.com/Alfresco/acs-deployment/blob/99635eacb1601236f174d42c0710e44a45662af6/helm/alfresco-content-services/templates/pvc.yaml)... Please fix as doing a review is too difficult currently

Reply:
fixed